PD-PiotrKurzydym/praca-domowaJS-nr3-Piotr-Kurzydym (toDoApp)/src/components/AllTasksHP.js

```diff
@@ -0,0 +1,16 @@
+const AllTasksHP = ({ items }) => {
+  const arrayHighPriority = items.filter(function (item) {
+    return item.priority === "high";
+  });
+
+  return arrayHighPriority.map((item) => (
```

Comment:
Lepiej zwracać jeden element z komponentu. Czyli pętla powinna być przynajmniej we fragmencie.